python/lsst/pipe/tasks/postprocess.py

```diff
@@ -1524,7 +1524,7 @@ def run(self, visitSummaryRefs):
                 visitSummary["psfSigma"] * visitSummary["pixelScale"] * np.sqrt(8 * np.log(2))
             )
             ccdEntry["skyRotation"] = visitInfo.getBoresightRotAngle().asDegrees()
-            ccdEntry["expMidpt"] = np.datetime64(visitInfo.getDate().toPython(), "ns")
+            ccdEntry["expMidpt"] = np.datetime64(visitInfo.date.nsecs(scale=dafBase.DateTime.TAI), "ns")
```

Comment:
So we have Python `datetime`, Numpy `datetime64`, Astropy `Time`, and our own `lsst.daf.base.DateTime`?  Ugh.